app/models/bishop.rb

```diff
@@ -1,7 +1,10 @@
 class Bishop < ChessPiece
-  def valid_move?(coordinates)
+  def valid_move?(coordinates, skip_checkstate_check = false)
```

Comment:
Not off at all. :+1: 

Called on ln 102 of `game.rb` - we reasoned that a *valid* move should not, by definition, include a move that would put the king into check. The flag is there so that, when we check if each *opponent* would have a valid move against the king, we don't "recursively" run the `move_puts_king_in_check?` on the opponent pieces (as it is not their turn).

Make sense?